frontend/src/components/Pages/FrontPage/MissionOverview/FailedMissionAlertView.tsx

```diff
@@ -47,32 +50,39 @@ interface MissionsProps {
 }
 
 function FailedMission({ mission }: MissionProps) {
-    let navigate = useNavigate()
-    const goToMission = () => {
-        let path = `${config.FRONTEND_BASE_ROUTE}/mission/${mission.id}`
-        navigate(path)
-    }
+    const missionLink = `${config.FRONTEND_BASE_ROUTE}/mission/${mission.id}`
 
     return (
-        <Button as={Typography} onClick={goToMission} variant="ghost" color="secondary">
-            <strong>'{mission.name}'</strong> {Text('failed on robot')} <strong>'{mission.robot.name}':</strong>{' '}
-            {mission.statusReason}
-        </Button>
+        <>
+            <HorizontalText>
+                <Typography link href={missionLink}>
+                    <strong>'{mission.name}'</strong>
+                </Typography>
+                <Typography color="secondary">{Text('failed on robot')}</Typography>
+                <Typography color="secondary">
+                    <strong>'{mission.robot.name}':</strong>
+                </Typography>
```

Comment:
I think the old design looks better, without the blue underscored text

Old:
![image](https://user-images.githubusercontent.com/33518988/229142917-a2c78a7b-2002-44fe-99ae-db33f680f016.png)

New:
![image](https://user-images.githubusercontent.com/33518988/229142990-25f8c23d-640c-4af1-be4a-c0eb656255cb.png)

Reply:
Reversed!